src/connector/src/parser/mod.rs

```diff
@@ -493,6 +495,142 @@ pub enum SpecificParserConfig {
     Bytes,
 }
 
+#[derive(Debug, Clone, Default)]
+pub struct ParserClientConfigList {
+    username: Option<String>,
+    password: Option<String>,
```

Comment:
Is it just for schema registry?

Reply:
Yes

Reply:
I think better name it `SchemaRegistryAuth`